.github/workflows/test-phpunit.yml

```diff
@@ -122,6 +122,10 @@ jobs:
         env:
           COMPOSER_AUTH: ${{ secrets.COMPOSER_AUTH }}
 
+      - name: Profile slow tests in PHP 7.4 MySQLi for now
+        if: matrix.php-versions == '7.4' && matrix.db-platforms == 'MySQLi'
+        run: echo "TACHYCARDIA_MONITOR_GA=enabled" >> $GITHUB_ENV
```

Comment:
It looks like the extension is enabled by default and I don't see your configuration disabling it anywhere. Why this explicit `enabled`?

Reply:
This env variable is for enabling the reporting in Github Actions as it is disabled by default.

Reply:
Is that just the `tabulate` or the entire extension? That was not clear to me from the docs. I added this to my toolkit (https://github.com/tattersoftware/codeigniter4-tools/commit/42124c8e18c7ba0499aa6539a760d67ad7c57242) but skipped the env Config because I thought it would run by default (with tabulate true).

Reply:
`tabulate: true` would just turn the console report from plain listing to tabular format.

The extension is handled by two env variables: `TACHYCARDIA_MONITOR` for console reporting (enabled by default) and `TACHYCARDIA_MONITOR_GA` for Github Actions reporting (disabled by default). The `tabulate` option is used only by the console reporting to modify the report's format. For GA, we use the workflow command syntax for annotating.

Reply:
Thanks for explaining. I see you added docs too! I will update my toolkit. For CI4, I think this PR is ready, unless you have more changes to commit?